docs/sources/installation/debian.md

```diff
@@ -28,9 +28,10 @@ To install the latest Debian package (may not be the latest Docker release):
 
 To verify that everything has worked as expected:
 
-    $ sudo docker run -i -t ubuntu /bin/bash
+    $ sudo docker run --rm hello-world
 
-Which should download the `ubuntu` image, and then start `bash` in a container.
+which should download the `hello-world` image, and then print a friendly Hello
```

Comment:
Thanks for the suggestion.

The hello-world container does not contain any information about the local Docker installation, so I will tweak that wording.